py3status/modules/spotify.py

```diff
@@ -102,6 +113,24 @@ def _get_text(self):
                 self.format_down,
                 self.py3.COLOR_OFFLINE or self.py3.COLOR_BAD)
 
+    def _compile_re(self, expression):
+        """
+        Compile given regular expression for current sanitize words
+        """
+        expression = expression.replace('META_WORDS_HERE', self.sanitize_words)
+        return re.compile(expression, re.IGNORECASE)
+
+    def _sanitize_title(self, title):
+        """
+        Remove redunant meta data from title and return it
+        """
+        inside_brackets = self._compile_re(r"([\(\[][^)\]]*?(META_WORDS_HERE)[^)\]]*?[\)\]])")
+        after_delimiter = self._compile_re(r"([\-,;/])([^\-,;/])*(META_WORDS_HERE).*")
```

Comment:
You could just compile these once in `post_config_hook()` rather than every time they get used.

I'm interested in your regexs it might be worth commenting on what they are doing eg.  remove content in parenthesis if one of sanitize_words is inside.

I note you don't care about matching so `...( featuring ]..` will be removed.  This is probably not a problem.

You can also end up with trailing whitespace from what I can see.